pkg/cli/snapshot.go

```diff
@@ -60,6 +61,10 @@ func copyExport() error {
 		if !forceOverride {
 			return fmt.Errorf("snapshot %s already exists; use -f to overwrite", snapshotFile)
 		}
+		err := os.RemoveAll(snapshotFile)
```

Comment:
You should use Remove not RemoveAll here

Reply:
@hemarkus changed it